scalability_and_performance/optimization/optimizing-networking.adoc

```diff
@@ -10,18 +10,18 @@ xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#
 
-Geneve encapsulates all tunneled traffic in user datagram protocol (UDP) packets. However, this leads to increased CPU utilization. Both these outer- and inner-packets are subject to normal checksumming rules to guarantee data is not corrupted during transit. Depending on CPU performance, this additional processing overhead can cause a reduction in throughput and increased latency when compared to traditional, non-overlay networks.
+Cloud, virtual, and bare-metal environments running {product-title} can use a high percentage of a NIC's capabilities with minimal tuning. Production clusters using OVN-Kubernetes with Geneve tunneling can effectively handle high-throughput traffic effectively and scale both up (for example, utilizing 100 Gbps NICs) and out (for example, adding more NICs) without requiring special configuration.
 
-Cloud, VM, and bare metal CPU performance can be capable of handling much more than one Gbps network throughput. When using higher bandwidth links such as 10 or 40 Gbps, reduced performance can occur. This is a known issue in Geneve-based environments and is not specific to containers or {product-title}. Any network that relies on Geneve or VXLAN tunnels will perform similarly because of the tunnel implementation.
+In some high-performance scenarios where maximum efficiency is critical, targeted performance tuning can help optimize CPU usage, reduce overhead, and ensure that you are making full use of the NIC's capabilities.
 
-If you are looking to push beyond one Gbps, you can:
+For environments where maximum throughput and CPU efficiency are critical, you can further optimize performance with the following strategies:
 
-* Evaluate network plugins that implement different routing techniques, such as border gateway protocol (BGP).
-* Use Geneve-offload capable network adapters. Geneve-offload moves the packet checksum calculation and associated CPU overhead off of the system CPU and onto dedicated hardware on the network adapter. This frees up CPU cycles for use by pods and applications, and allows users to utilize the full bandwidth of their network infrastructure.
+* Validate network performance using tools such as `iPerf3` and `k8s-netperf`. These tools allow you to benchmark throughput, latency, and packets-per-second (PPS) across pod and node interfaces.
+
+* Evaluate OVN-Kubernetes User Defined Networking (UDN) routing techniques, such as border gateway protocol (BGP).
 
-Geneve-offload does not reduce latency. However, CPU utilization is reduced even in latency tests.
+* Use Geneve-offload capable network adapters. Geneve-offload moves the packet checksum calculation and associated CPU overhead off of the system CPU and onto dedicated hardware on the network adapter. This frees up CPU cycles for use by pods and applications, and allows users to utilize the full bandwidth of their network infrastructure.
 
-// Optimizing the MTU for your network
```

Comment:
Any reason to remove this? Comments with the module name can be helpful for maintenance

Reply:
I don't think so.  Mis-matched MTUs can certainly degrade performance, so it seems reasonable to have it in that bulleted "strategies" section at the end. Its also a nice lead-in into the next section on MTU.